be/src/olap/rowset/alpha_rowset_writer.cpp

```diff
@@ -200,6 +200,13 @@ RowsetSharedPtr AlphaRowsetWriter::build() {
     _current_rowset_meta->set_num_rows(_num_rows_written);
     _current_rowset_meta->set_creation_time(time(nullptr));
 
+    // validate rowset arguments before create rowset
+    bool ret = _validate_rowset();
+    if (!ret) {
+        LOG(WARNING) << "valiate rowset arguments failed";
+        return nullptr;
```

Comment:
Just fatal here， because the caller may not check rowset == nullptr, and validate == false is an exception, should not happen.